android/src/main/java/com/reactnativehmssdk/HMSRNSDK.kt

```diff
@@ -506,6 +510,13 @@ class HMSRNSDK(
               networkQualityUpdatesAttached = false
               rtcStatsAttached = false
               callback?.resolve(emitHMSSuccess())
```

Comment:
callback should be in else of `fromPIP` if check

Reply:
done in commit - 24c6abacc4d59074060b1c0467918f53b6f27165